00 - SOCSim - IPS Alert Triage/A: Get Nessus Assets.json

```diff
@@ -0,0 +1,136 @@
+{
```

Comment:
Please name the collection as '02 - UseCase - IPS Alert Triage'
All Use Cases wil be coming at 02 sequence